resources/styles/components/scores/index.less

```diff
@@ -64,23 +70,63 @@
     }
     &.is-ascending:after  { content: @fa-var-sort-asc;  }
     &.is-descending:after { content: @fa-var-sort-desc; }
+
+    &[data-assignment-type] {
+      height: 100%;
+    }
+
+    .tutor-plan-set(heading);
+  }
+
+  .scores-cell, .assignment-header-cell, .student-name {
+    width: 100%;
+    height: 100%;
+    display: block;
+  }
+
+  .scores-cell, .assignment-header-cell {
+    .tutor-plan-set(cell);
   }
 
   i.late {
     .tutor-late-icon();
     padding-left:8px;
   }
+  .public_fixedDataTable_header, .public_fixedDataTable_header .public_fixedDataTableCell_main {
+    background: @tutor-white;
+  }
   .assignment-header-cell {
-    background: @tutor-neutral-light;
-    time {
-      &:before {
-        content: "Due:";
-        margin-right: 0.5rem;
+    padding: 0;
+
+    > div {
+      display: block;
+      width: 100%;
+      height: 50%;
+      padding: 8px;
+    }
+
+    > div:nth-child(even) {
+      background-color: #f6f7f8;
+    }
+
+    &[data-assignment-type] {
+      font-weight: 400;
+
+      > div:nth-child(even) {
+        mix-blend-mode: overlay;
       }
     }
+
     .review-plan {
-      float: right;
-      margin-right: 1rem;
+      &:not(.btn-link):not(.btn-flat):not(.btn-fab).btn {
```

Comment:
this i'd like to do in a separate PR, we have this wonky selector stuff with buttons elsewhere as well, and it needs to be fixed